tests/lib/eslint/eslint.js

```diff
@@ -928,6 +928,7 @@ describe("ESLint", () => {
 
             it("should throw if eslint.config.js file is not present even if overrideConfig was passed", async () => {
                 eslint = new ESLint({
+                    flags,
```

Comment:
It looks like this option was accidentally omitted here and in other unit tests in #18742.